src/spikeinterface/core/base.py

```diff
@@ -798,7 +799,10 @@ def load(file_path: Union[str, Path], base_folder: Optional[Union[Path, str, boo
             return extractor
 
         else:
-            raise ValueError("spikeinterface.Base.load() file_path must be an existing folder or file")
+            error_msg = f"Failed to load the file_path {file_path} must be an existing folder or file"
```

Comment:
```suggestion
            error_msg = f"Failed to load the file_path {file_path}. The file_path must be an existing folder or file"
```

Or something like this. You failed to load the file_path `.` (why?) because it must be an existing folder or file.

Reply:
Yes, this should be corrected.

Concerning of how to make the information of `base_folder` better I agree and thought about it before. But I honestly don't know how to improve this. The specification of how to use `base_folder` is sort of implicit and left to the user. People used `relative_to` to save with respect to this folder and then need to correct their paths with respect to this and pass it like this.

Any ideas on how to make this better are welcomed but I think perfect should not be the enemy of good here.

Btw, I believe that warnings should be actionable. I am not sure I believe that about errors but it makes sense as well as an ideal. I will think about it.

Reply:
In that case it seems to me like the message should be something like

```
The provided base_folder is {base_folder}. Please ensure this is the correct root for the relative paths  to your extractor
```

or something along that vein. I don't know this part of the code well enough to know for sure this is the correct statement. But this reminds the user that the base_folder is given for the relative_to machinery. 

> Btw, I believe that warnings should be actionable. I am not sure I believe that about errors but it makes sense as well as an ideal. I will think about it.

I stand corrected. But in my schema a ValueError, TypeError, FileNotFoundError are things that ought to be fixable right. So if we can nudge the person to fix it that should be the goal, I think. So warning + those would be my metric. Something like a RunTimeError for me says nope this is fundamentally wrong.

Reply:
> or something along that vein. I don't know this part of the code well enough to know for sure this is the correct statement. 

We have the same problem.

Ok. I think I am removing this part. Maybe that error should be deeper and I need to think better about relative paths. About every 3 months I try to think on the relative_path machinery and I never feel that I completly grok it.

Your heuristics about those type of errors makes sense. Maybe less clear about file not found. You can say what is the problem but you don't know if the file does no exists, is in the wrong place, is corrupted, wrong extension, wrong mount, etc.

Reply:
Yeah, then I would remove it. Because technically the user can just press the up-arrow and see what they put in the `base_folder` argument.

For me filenotfound implies all those things and the user needs to figure out which, but maybe that is expecting too much.

But yeah one of these days you and I can have a zoom and just stare at the code together until we both understand it :)

Reply:
Ok, removed this.